core/services/offchainreporting/peerstore.go

```diff
@@ -14,22 +14,23 @@ import (
 	"github.com/pkg/errors"
 	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/services/postgres"
+	"github.com/smartcontractkit/chainlink/core/store/models"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
 type (
 	P2PPeer struct {
 		ID        string
 		Addr      string
-		JobID     int32
+		PeerID    string
```

Comment:
Is the thinking here that the same node could have multiple peerIDs? Or just data denormalization

Reply:
Yes one node could have multiple peer IDs